android/app/src/main/java/com/lndr/PayPalModule.java

```diff
@@ -147,6 +155,8 @@ public void sendPayPalPayment(Double amount, String currencyCode, String payeeEm
         PayPalPayment.PAYMENT_INTENT_SALE
       );
 
+    payment.payeeEmail(payeeEmail);
```

Comment:
Can we do some parameter checking in this function prior to creating the PayPalPayment object to make sure the passed in parameters have lengths > 0 ?

Reply:
yes, and I would prefer to do the parameter checking on the JS level